nana/modules/cat.py

```diff
@@ -0,0 +1,37 @@
+from pyrogram import filters
+from pyrogram.types import InputMediaPhoto, InlineKeyboardMarkup, InlineKeyboardButton
+from nana import setbot, AdminSettings, BotUsername, app, Command
+from nana.helpers.PyroHelpers import ReplyCheck
+import speedtest
+import re
+import random
+
+def cat_callback(_, __, query):
+    if re.match("cat_pic", query.data):
+        return True
+
+cat_create = filters.create(cat_callback)
+
+def speed_convert(size):
+    """Hi human, you can't read bytes?"""
+    power = 2 ** 10
+    zero = 0
+    units = {0: '', 1: 'Kb/s', 2: 'Mb/s', 3: 'Gb/s', 4: 'Tb/s'}
+    while size > power:
+        size /= power
+        zero += 1
+    return f"{round(size, 2)} {units[zero]}"
+
+@setbot.on_callback_query(cat_create)
+async def catpic_callback(client, query):
+    if query.from_user.id in AdminSettings:
+        image = f"https://d2ph5fj80uercy.cloudfront.net/0{random.randint(1, 6)}/cat{random.randint(0,4999)}.jpg"
```

Comment:
Codacy found an issue: [Standard pseudo-random generators are not suitable for security/cryptographic purposes.](https://app.codacy.com/gh/MadeByThePinsHub/Nana-Remix/pullRequest?prid=6330176)